decidim-admin/spec/features/manage_participatory_processes_spec.rb

```diff
@@ -37,12 +37,12 @@
       expect(page).to have_content(participatory_process.subtitle["en"])
       expect(page).to have_content(participatory_process.subtitle["es"])
       expect(page).to have_content(participatory_process.subtitle["ca"])
-      expect(page).to have_content(participatory_process.short_description["en"])
-      expect(page).to have_content(participatory_process.short_description["es"])
-      expect(page).to have_content(participatory_process.short_description["ca"])
-      expect(page).to have_content(participatory_process.description["en"])
-      expect(page).to have_content(participatory_process.description["es"])
-      expect(page).to have_content(participatory_process.description["ca"])
+      expect(page).to have_content(participatory_process.short_description["en"].gsub(/<\/?[^>]*>/, ""))
```

Comment:
Also, the `gsub` is a simple way to remove the HTML keys from the string (I know it's not really bullet-proof, but we don't need that much here).